RussianFilter/sections/general_extensions.txt

```diff
@@ -8,6 +8,8 @@ torren.eu#%#//scriptlet('remove-attr', 'onclick', 'a[download]')
 isport.ua#%#//scriptlet("abort-current-inline-script", "decodeURIComponent", "delete window")
 ! https://github.com/AdguardTeam/AdguardFilters/issues/67949
 93.88.74.236#%#//scriptlet("abort-on-property-read", "open")
+! https://github.com/AdguardTeam/AdguardFilters/issues/70650
+9hentai.ru#%#//scriptlet("prevent-window-open")
```

Comment:
Unfortunately, it doesn't work on my end. Please check this one `9hentai.ru#%#//scriptlet("prevent-addEventListener", "click", "flashElement")`. The wrong filter is also selected.